bytes/bytes/database/db.py

```diff
@@ -11,11 +11,18 @@
 
 
 @lru_cache(maxsize=1)
-def get_engine(db_uri: str) -> Engine:
-    logger.info("Connecting to database..")
+def get_engine(db_uri: str, pool_size: int) -> Engine:
```

Comment:
I don't see a reason pool_size should be a function argument here, we can just get from the setting directly from the settings object like it is done in boefjes and mula. That would also fix the failing tests.

Reply:
I updated the tests, and created a separate issue https://github.com/minvws/nl-kat-coordination/issues/2550